src/openvic-simulation/pop/Pop.cpp

```diff
@@ -206,7 +418,7 @@ bool PopManager::load_pop_type_file(
 		"issues", ZERO_OR_ONE, assign_variable_callback(issues_node),
 		"demote_migrant", ZERO_OR_ONE, expect_bool(assign_variable_callback(demote_migrant)),
 		"administrative_efficiency", ZERO_OR_ONE, expect_bool(assign_variable_callback(administrative_efficiency)),
-		"tax_eff", ZERO_OR_ONE, success_callback, // TODO - tax collection modifier
+		"tax_eff", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(tax_efficiency)),
 		"can_build", ZERO_OR_ONE, expect_bool(assign_variable_callback(can_build)),
```

Comment:
can_build is somewhat misleading, this determines whether the pop can invest into factories & railroads (depending on economic policy)
is_capitalist? (might be too strict)
is_private_investor? (might be confused with shadowy financeers)
What do you think is a good name?
maybe can_build is fine. Just think about it for a second and see if you know a better name.

Reply:
I'll change it to `can_invest`